releases.md

```diff
@@ -0,0 +1,106 @@
+# Express Release Process
+
+This document contains the technical aspects of the Express release process. The
+intended audience is those who have been authorized by the Express Technical
+Committee (TC) to create, promote and sign official release builds for Express,
+as npm packages hosted on https://npmjs.com/package/express.
+
+## Who can make releases?
+
+Release authorization is given by the Express TC. Once authorized, an individual
+must have the following:
+
+### 1. Github release access
+
+Releases can be tagged by members of the @expressjs/express team with Write or
+Admin permission level.
+[tunniclm: Is there  a separate way to indicate whether an individual is allowed
```

Comment:
I would say it should be tagged by the person who is building the `tar.gz` package for npm.